src/main/java/com/example/onlinebookstore/dto/BookRequestDto.java

```diff
@@ -1,19 +1,26 @@
 package com.example.onlinebookstore.dto;
 
+import jakarta.validation.constraints.Min;
+import jakarta.validation.constraints.NotNull;
 import java.math.BigDecimal;
 import lombok.Data;
 
 @Data
-public class CreateBookRequestDto {
+public class BookRequestDto {
+    @NotNull
     private String title;
-    
+
+    @NotNull
     private String author;
-    
+
+    @NotNull
+    @Min(value = 0, message = "can't be less than 0")
     private BigDecimal price;
-    
+
+    @NotNull
     private String isbn;
```

Comment:
The ISBN value is unified and consists of 13 digits. I think it will be good to add a check for this.